app/services/DepartureP5MessageService.scala

```diff
@@ -63,17 +72,31 @@ class DepartureP5MessageService @Inject() (
           message =>
             message.latestMessage.messageType match {
               case RejectedByOfficeOfDeparture =>
-                isErrorAmendable(movement.departureId, message.latestMessage.messageId, movement.localReferenceNumber).map {
-                  case (rejectionType, isDeclarationAmendable, xPaths) =>
-                    RejectedMovementAndMessage(
-                      movement.departureId,
-                      movement.localReferenceNumber,
-                      movement.updated,
-                      message,
-                      BusinessRejectionType(rejectionType),
-                      isDeclarationAmendable,
-                      xPaths
+                isPrelodgeError(movement.departureId, message.latestMessage.messageId).flatMap {
```

Comment:
Very very minor but the code might read slightly better if we have a method that returns just the rejectionType and xPaths, then we match on the rejection type

```
message.latestMessage.messageType match {
  case RejectedByOfficeOfDeparture =>
    getRejectionMessage(movement.departureId, message.latestMessage.messageId).flatMap {
      case (PresentationNotificationRejection.value, xPaths) =>
                    Future(
                      PrelodgeRejectedMovementAndMessage(
                        movement.departureId,
                        movement.localReferenceNumber,
                        movement.updated,
                        message,
                        BusinessRejectionType(rejectionType),
                        xPaths
                      )
                    )
                  case (rejectionType, xPaths) =>
                        ...
```

Also, any need for PrelodgeRejectedMovementAndMessage to contain the rejectionType param?